examples/kitchen-sink.yaml

```diff
@@ -261,6 +270,15 @@ meter_provider:
             # Configure default histogram aggregation. Values include: explicit_bucket_histogram, base2_exponential_bucket_histogram. For behavior of values, see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md.
             # If omitted or null, explicit_bucket_histogram is used.
             default_histogram_aggregation: base2_exponential_bucket_histogram
+    - # Configure a periodic metric reader.
+      periodic:
+        # Configure exporter.
+        exporter:
+          # Configure exporter to be OTLP with file transport.
+          otlp_file:
+            # Configure output stream.
+            # If omitted or null, stdout is used.
+            output_stream: metrics.jsonl
```

Comment:
For completeness of the example, add `temporality_preference` and `default_histogram_aggregation` ?

This is the kitchen sink, it should show all settings.